src/awscdk/base.ts

```diff
@@ -366,14 +375,15 @@ ${appCode}
    * and destroying the stack when no longer needed.
    */
   protected createFeatureStage() {
+    const stackId = this.baseOptions.deploySubStacks ? `${this.stackPrefix}-feature ${this.stackPrefix}-feature/*` : `${this.stackPrefix}-feature`;
```

Comment:
shouldnt this be a global variable initiated in the constructor?

Reply:
It depends on each stage -feature -personal -dev etc

Reply:
Then "thats" the part that goes in here and the only shit you do would be an apply format?

you could even pull this out into a method `getStackId(stage)` instead...